applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -100,6 +110,15 @@ def __init__(self, model, custom_settings):
 
         self._bfecc_convection = self.settings["bfecc_convection"].GetBool()
 
+        # this is used to perform one step BFECC for the eulerian LS convection
+        self._eulerian_error_compensation = self.settings["levelset_convection_settings"]["eulerian_error_compensation"].GetBool()
+
+        # this is used to identify the splitting of LS convection (Strang splitting idea)
+        self._levelset_splitting = self.settings["levelset_convection_settings"]["levelset_splitting"].GetBool()
+        dt_factor = 0.5 if self._levelset_splitting else 1.0
+        self._levelset_dt_factor = dt_factor
+        self.main_model_part.ProcessInfo.SetValue(KratosMultiphysics.DELTA_TIME_FACTOR, dt_factor)
```

Comment:
This only needs to be done if the `self._levetset_splitting` is `True`.

Reply:
Done.